charts/datahub-executor-worker/values.yaml

```diff
@@ -89,3 +89,15 @@ readinessProbe:
   periodSeconds: 60
   failureThreshold: 3
   timeoutSeconds: 5
+
+persistentVolume:
+  enabled: false
+  accessModes:
+    - ReadWriteOnce
+  annotations: {}
+  labels: {}
+  existingClaim: ""
+  mountPath: /tmp/datahub
+  size: 100Gi
+  # storageClass: "-"
```

Comment:
what is an example value here?

do you thik their engineers will understand how to correctly configure to store on say S3?

Reply:
if that parameter is omitted, this will use default storage class configured in their cluster. I'd not go with S3 volumes, because that path is used by ingestions to store venv and logs. We actually use similar config in ECS (cloudformation).

Alternatively, they can mount it under a different path (e.g. /memory-profiles), and then use S3 volumes, but I'm not sure their kube cluster supports it. Usually EBS gp2/gp3 is the only supported volume type in AWS deployments.